Version prep

bassmanitram opened the pull request:
Release 1.4.0 prep

biokomiker commented:
Seems to work - nice to directly see the commands (and to learn from them) and eventually to import entries from others ;)

biokomiker commented:
A command I use to compress video files generated from my camera:
        {
            "type": "command",
            "label": "encode video with H.265",
            "command_line": "ffmpeg -y -hide_banner  -i %b  -vf yadif -c:v hevc  %w.mp4 ; exiftool -ee  -overwrite_original_in_place -tagsFromFile %b %w.mp4",
            "cwd": "%d",
            "use_shell": true,
            "mimetypes": [
                "video/*"
            ],
            "filetypes": [
                "!directory",
                "standard"
            ]
        },

It works with MTS files (Sony) and MOV files (Canon ?) which have H264 codec. Compression is ~ 10x 
I added the exiftool command to copy the tag of recorded time stamp.

bassmanitram commented:
> It works with MTS files (Sony) and MOV files (Canon ?) which have H264 codec. Compression is ~ 10x
> I added the exiftool command to copy the tag of recorded time stamp.

Very cool - thx. I've added it to the wiki.

Can you send me the JSON config for your deja-dup action too. 

Thx
